### ISSUE TITLE
Action failing with out of range array

### ISSUE THREAD

**Issue by jpacareu-meli** [Bhupesh-V/memer-action#15]
**Describe the bug**
Hi I'm having this error quite a lot with the action.

![image](https://user-images.githubusercontent.com/57413670/88837058-b124a800-d1a5-11ea-9e9c-adc41a88a0c4.png)

Is there any chance that we have a fallback meme URL in case default one fails?

**Comment by Bhupesh-V** [comment]
Hi @jpacareu-meli 👋🏽 
Thanks for reporting this. I too have started having this problem now 😅 
I think a fallback meme would be a Good Idea moreover we can provide input fields for this

```yml
uses: Bhupesh-V/memer-action@master
with:
    filter: "new"
    default_meme: ""
    default_source: ""
    default_title: ""

```
What do you think?

**Comment by jpacareu-meli** [comment]
Excelent solution 💪

**Comment by Bhupesh-V** [comment]
Fixed in [0.3.0](https://github.com/Bhupesh-V/memer-action/releases/tag/0.3.0)
